ext/http/01_http.js

```diff
@@ -135,22 +135,28 @@ class HttpConn {
       body !== null ? new InnerBody(body) : null,
       false,
     );
-    const signal = abortSignal.newSignal();
+    const abortController = new AbortController();
     const request = fromInnerRequest(
       innerRequest,
-      signal,
+      abortController.signal,
       "immutable",
       false,
     );
 
-    const respondWith = createRespondWith(
+    const responseHandler = createRespondWith(
       this,
       streamRid,
       request,
       this.#remoteAddr,
       this.#localAddr,
     );
 
+    const respondWith = (resp) =>
+      responseHandler(resp).catch((error) => {
```

Comment:
This bit should use a primordial - instead of `.catch()` it should be `PromisePrototypeCatch`

Reply:
Actually I think you could just pass `abortController` to `createRespondWith` and add a `catch` block there

Reply:
Good idea, that's a lot cleaner 👍